matlab_simulation/05-estimation/kalman_filter/measurement_model.m

```diff
@@ -0,0 +1,53 @@
+function [C,D,Q,m] = measurement_model(example)
```

Comment:
I agree that it will be more difficult to change the examples having these motion models and measurement models as functions, but one of the requirement in this issue is that "examples need to be easily selectable," so I guess that's the trade-off.